tox.ini

```diff
@@ -25,9 +25,9 @@ commands =
 	pre-commit run --all-files --show-diff-on-failure {posargs:}
 
 [testenv:update_dependencies]
-description = update requirements.txt
+description = update lockfile
 skip_install = true
 deps =
-    pip-tools
+    uv
 commands =
-    pip-compile --strip-extras requirements.in {posargs:}
```

Comment:
I think we can removed this completely: it's in the contributing.md and otherwise gives us 2 ways of doing the same thing (once with uv and another with tox that calls uv)